lib/wanda/catalog/customization_policy.ex

```diff
@@ -0,0 +1,19 @@
+defmodule Wanda.Catalog.CustomizationPolicy do
+  @moduledoc """
+  Policy for checks customizations.
+  """
+  @behaviour Bodyguard.Policy
+
+  import Wanda.Support.AbilitiesHelper
+
+  alias Wanda.Catalog.CheckCustomization
+  alias Wanda.Users.User
+
+  def authorize(:apply_custom_values, %User{} = user, CheckCustomization),
```

Comment:
We need to add the `defdelegate` in the `check_customizations.ex` file:
https://hexdocs.pm/bodyguard/readme.html#policies
Example: https://github.com/trento-project/web/blob/main/lib/trento/users/user.ex#L25

It works without it in some cases, but better to simply add it